harbor/assets/service_checks.json

```diff
@@ -0,0 +1,92 @@
+[
+    {
+        "agent_version": "6.13.0",
+        "integration": "harbor",
+        "groups": ["host"],
+        "check": "harbor.status",
+        "statuses": ["ok", "critical"],
+        "name": "Overall status",
+        "description": "Returns `OK` if the Harbor API is reachable and answers correctly, `CRITICAL` if connection is not possible or if the API says the registry is unhealthy."
+    },
+    {
+        "agent_version": "6.13.0",
+        "integration": "harbor",
+        "groups": ["host"],
+        "check": "harbor.component.chartmuseum.status",
+        "statuses": ["ok","critical"],
+        "name": "Status of the `Chartmuseum` service",
+        "description": "Service checks is not emitted if the chartmuseum is not configured. Returns `OK` if the service is healthy, `CRITICAL` otherwise."
```

Comment:
```suggestion
        "description": "Returns `OK` if the service is healthy, otherwise returns `CRITICAL`. This service check is not emitted unless the chartmuseum is configured."
```